server/bundles/io.cloudbeaver.service.auth/schema/service.auth.graphqls

```diff
@@ -139,8 +144,13 @@ extend type Query {
     authUpdateStatus(authId: ID!, linkUser: Boolean): AuthInfo!
 
     # Logouts user. If provider not specified then all authorizations are revoked from session.
+    @deprecated
     authLogout(provider: ID, configuration: ID): Boolean
 
+    # Same as #authLogout, but returns additional information
+    @since(version: "23.3.3")
+    authLogoutExtended(provider: ID, configuration: ID): LogoutInfo!
```

Comment:
it's not clear what `Extended` suffix means especially if we will remove `authLogout`

Reply:
Alexander decided `authLogout` function to be for a while for back backward compatibility. so this is why we have new function with `Extended` suffix

Reply:
problem is that `Extended` means nothing this is not good suffix for api
`authLogoutInfo` sound better